src/azure-cli/azure/cli/command_modules/vm/_help.py

```diff
@@ -91,38 +91,31 @@
     crafted: true
 """
 
-helps['disk-encryption-set create'] = """
+helps['disk-access create'] = """
```

Comment:
Please pull the latest code from the dev branch. The help messages of `disk-access` command group have been removed.

Reply:
Updated and removed